erpnext/stock/report/stock_analytics/stock_analytics.py

```diff
@@ -144,7 +144,8 @@ def get_periodic_data(entry, filters):
 		# if period against item does not exist yet, instantiate it
 		# insert existing balance dict against period, and add/subtract to it
 		if periodic_data.get(d.item_code) and not periodic_data.get(d.item_code).get(period):
-			periodic_data[d.item_code][period] = periodic_data[d.item_code]['balance']
+			previous_balance = periodic_data[d.item_code]['balance'].copy()
```

Comment:
Should this be a deepcopy? `balance` also seems to be a nested dict.

Reply:
Ignore. It's just a WH -> number mapping.

Reply:
@ankush its not 
![Screenshot 2021-08-13 at 3 38 24 PM](https://user-images.githubusercontent.com/25857446/129341354-12416bd3-902b-4a04-b266-298cd514b77e.png)